plugins/inputs/http_response/http_response.go

```diff
@@ -198,6 +199,10 @@ func (h *HTTPResponse) httpGather(u string) (map[string]interface{}, map[string]
 		return nil, nil, err
 	}
 
+	if _, uaPresent := h.Headers["User-Agent"]; !uaPresent {
+		request.Header.Set("User-Agent", "telegraf/"+internal.Version+" (+https://github.com/influxdata/telegraf)")
```

Comment:
Thanks for the PR - in other plugins we use internal.ProductToken(), any objection to using that rather than having another custom setting?

Reply:
Oh I didn't notice this - yeah that looks much better! Will change...